src/packages/utils/src/things/json-rpc/json-rpc-quantity.ts

```diff
@@ -13,8 +14,9 @@ export class Quantity extends BaseJsonRpcType {
 
   _nullable: boolean = false;
 
-  public static from(value: JsonRpcInputArg, nullable = false) {
-    if (value instanceof Quantity) return value;
+  public static from(value: JsonRpcInputArg, nullable = false): Quantity {
+    if (Quantity.isQuantity(value))
+      return value;
```

Comment:
This actually can't happen (per the types). Can we remove this check altogether?

```suggestion
```

Reply:
That's true, but we do use this in the `persistent-cache` key generation. 

Blocknumber is passed in as either string or Quantity instance. I've pushed that differentiation back out into `persistent-cache`, so that the `Quantity` implementation is cleaner.